slick.dataview.js

```diff
@@ -511,14 +513,26 @@
     }
 
     function expandCollapseAllGroups(level, collapse) {
-      if (level == null) {
+      if (level === null) {
```

Comment:
hmm I see you've put all equality with triple equal but I wonder if this one in particular will break something. If the `level` can be `undefined`, then this will be broken